thirdservice/src/main/java/com/services/thirdservice/db/repositories/user/UserRepository.java

```diff
@@ -1,11 +1,51 @@
 package com.services.thirdservice.db.repositories.user;
 
-import org.springframework.data.repository.CrudRepository;
+import java.io.IOException;
+import java.util.Arrays;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Repository;
 
+import com.google.api.services.sheets.v4.Sheets;
+import com.google.api.services.sheets.v4.model.AppendValuesResponse;
+import com.google.api.services.sheets.v4.model.ValueRange;
 import com.services.thirdservice.db.entity.user.User;
 
-@Repository
-public interface UserRepository extends CrudRepository<User, Integer> {
+import lombok.extern.slf4j.Slf4j;
 
+@Repository
+@Slf4j
+public class UserRepository {
+    
+    @Value("${google.spreadsheet.id}")
+    private String spreadsheetId;
+    
+    @Autowired
+    private Sheets sheets;
+    
+    public String create(User user) throws IOException {
+        String range = "test!A1:C1";
+        ValueRange body = new ValueRange()
+                .setValues(Arrays.asList(
+                        Arrays.asList(user.getFirstName())));
+        AppendValuesResponse response = sheets.spreadsheets().values().append(spreadsheetId, range, body)
+                .setValueInputOption("USER_ENTERED")
+                .setInsertDataOption("OVERWRITE")
+                .setIncludeValuesInResponse(true)
+                .execute();
+//////////////////DO NOT TOUCH, magic//////////////////////////////
```

Comment:
You should rewrite this code, as everyone understands what is going on here. So check it again and left more informative comment